app/views/posts/index.html.erb

```diff
@@ -8,9 +8,11 @@
     <%# portrait vs landscape?? %>
     
   <% end %>
-  <p><%= post.message %></p>
-<% end %>
+  <h4><%= post.message %></h4>
+  <p id="likes<%=post.id%>">Likes: <%= post.likes.count %></p>
```

Comment:
Likes shouldn't be counted from likes table each time, should keep running tally in posts table and take from there (weird to have likes as an array).